dask/dataframe/multi.py

```diff
@@ -475,11 +480,12 @@ def merge(
         )
 
     # Single partition on one side
+    # Note that cudf supports "leftsemi" and "leftanti" joins
     elif (
         left.npartitions == 1
         and how in ("inner", "right")
         or right.npartitions == 1
-        and how in ("inner", "left")
+        and how in ("inner", "left", "leftsemi", "leftanti")
```

Comment:
Thanks @rjzamora! Small nitpick: could you create a variable which stores these values so we can reference it throughout the codebase (i.e. `how in allowed_hows`)

Reply:
Thanks for the review @jrbourbeau - Good suggestion.  Hopefully I addressed this.